xstate-chatbot/nodejs/src/machine/vitals.js

```diff
@@ -1010,18 +1021,31 @@ const vitalsFlow = {
     addVitals: {
       id: 'addVitals',
       invoke: {
-        src: (context) => vitalsService.addVitals(context.user, context.slots.vitals),
-        onDone: {
+        src: (context) => vitalsService.addVitals(context.user, context.slots.vitals,context.slots.registerPatient),
+        onDone: [
+        {
+          cond: (context, event) =>context.slots.person.rrt==='NO',
```

Comment:
Just remove cond from here. That will execute the action without checking for any condition

Reply:
For both condition dialog.get_message is different